src/prefect/agent/agent.py

```diff
@@ -34,16 +35,21 @@ class Agent:
 
     In order for this to operate `PREFECT__CLOUD__AGENT__AUTH_TOKEN` must be set as an
     environment variable or in your user configuration file.
+
+    Args:
+        - name (str, optional): An optional name to give this agent. Can also be set through
+            the environment variable `AGENT_NAME`. Defaults to "agent"
```

Comment:
I suggest PREFECT_AGENT_NAME to keep all configs immediately visible via `env | sort`. This won't get put in the config (although it could?) because there's no double underscore.

Reply:
It _could_ get put in the config but @cicdw and I had said offline that it doesn't feel like a needed config option. However it definitely could get put in if we think it would be a desired option!